src/ax/type.go

```diff
@@ -30,18 +30,25 @@ func TypeOf(m Message) MessageType {
 // TypeByName returns the message type for a fully-qualified Protocol Buffers
 // message name.
 //
-// If the message type is registered, mt is the message type of n, and ok is
-// true; otherwise, ok is false.
+// If the message type is registered and mt is the message type of `ax.Message`,
+// ok is true; otherwise, ok is false.
 //
```

Comment:
I think you've swapped docs for the parameter and the return value here. Perhaps say:

> If n is the name of a registered implementation of Message, then mt is the type of that message, and ok is true; otherwise, ok is false.